hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -1605,8 +1663,9 @@ bool MDMParser::getSignalStrength(NetStatus &status)
         MDM_INFO("\r\n[ Modem::getSignalStrength ] = = = = = = = = = =");
 
         // Ensure modem is responsive
-        if (!_atOk())
+        if (!_checkModem()) {
             goto cleanup;
+        }
```

Comment:
Could we add braces around the single line statement just below as well please? :D 
```
if (RESP_OK != waitFinalResp(nullptr, nullptr, COPS_TIMEOUT))
            goto cleanup;
```

Reply:
Let's do any unrelated refactoring separately.

Reply:
I always try to clean up at least within the scope of the function I'm currently making changes in.  If we wait to clean up later, it will likely never happen any time soon.

Reply:
No strong opinions here, but we have some refactoring tasks coming up anyway for example logging as well as https://github.com/particle-iot/device-os/pull/2198#discussion_r490504313.